File: sdkwork-backend-sdk-csharp/Models/GenerationContext.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Backend.Models
{
    public class GenerationContext
    {
        public int? GenerationId { get; set; }
        public string? GenerationType { get; set; }
        public string? GenerationStatus { get; set; }
        public string? Channel { get; set; }
        public string? RequestId { get; set; }
        public string? ChannelTaskId { get; set; }
        public string? RoutingStrategy { get; set; }
        public string? RoutingProduct { get; set; }
        public string? RoutingProvider { get; set; }
    }
}

File: sdkwork-backend-sdk-go/types/generation_context.go
package types


type GenerationContext struct {
	GenerationId int `json:"generationId"`
	GenerationType string `json:"generationType"`
	GenerationStatus string `json:"generationStatus"`
	Channel string `json:"channel"`
	RequestId string `json:"requestId"`
	ChannelTaskId string `json:"channelTaskId"`
	RoutingStrategy string `json:"routingStrategy"`
	RoutingProduct string `json:"routingProduct"`
	RoutingProvider string `json:"routingProvider"`
}
